src/views/virtualmachines/list/components/VMStatusConditionLabel/VMStatusConditionLabel.tsx

```diff
@@ -19,7 +19,7 @@ export const VMStatusConditionLabel: React.FC<V1VirtualMachineCondition> = React
             e.preventDefault();
           }}
         >
-          {condition?.reason}={condition?.status}
+          {condition?.reason}={condition?.type}
```

Comment:
we want to show the condition's type and the condition's status
so please change this line to `{condition?.type}={condition?.status}`